From 18ec2f246d142c849bc589c83c76445499cb7a64 Mon Sep 17 00:00:00 2001
From: Fancy Fang <chen.fang@nxp.com>
Date: Tue, 11 Sep 2018 15:26:39 +0800
Subject: [PATCH 4584/5242] MLK-19537-3 drm/imx: sec-dsim_imx: remove dphy
 slave reset

commit  3e92f5b21c5555c9e59ea16f971768549f772ea6 from
https://source.codeaurora.org/external/imx/linux-imx.git

Remove dphy slave reset and only keep master reset for mipi dsi

Signed-off-by: Fancy Fang <chen.fang@nxp.com>
Reviewed-by: Robby Cai <robby.cai@nxp.com>
(cherry picked from commit 83f4a5a95262184759f1c99be46e8906dd24543b)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/gpu/drm/imx/sec_mipi_dsim-imx.c |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/gpu/drm/imx/sec_mipi_dsim-imx.c b/drivers/gpu/drm/imx/sec_mipi_dsim-imx.c
index 548c356..2776f11 100644
--- a/drivers/gpu/drm/imx/sec_mipi_dsim-imx.c
+++ b/drivers/gpu/drm/imx/sec_mipi_dsim-imx.c
@@ -101,12 +101,12 @@ static void imx_sec_dsim_lanes_reset(struct regmap *gpr, bool reset)
 	if (!reset)
 		/* release lanes reset */
 		regmap_update_bits(gpr, GPR_MIPI_RESET_DIV,
-				   GPR_MIPI_S_RESETN | GPR_MIPI_M_RESETN,
-				   GPR_MIPI_S_RESETN | GPR_MIPI_M_RESETN);
+				   GPR_MIPI_M_RESETN,
+				   GPR_MIPI_M_RESETN);
 	else
 		/* reset lanes */
 		regmap_update_bits(gpr, GPR_MIPI_RESET_DIV,
-				   GPR_MIPI_S_RESETN | GPR_MIPI_M_RESETN,
+				   GPR_MIPI_M_RESETN,
 				   0x0);
 }
 
-- 
1.7.9.5

